baybe/campaign.py

```diff
@@ -159,7 +159,7 @@ def validate_config(cls, config_json: str) -> None:
     def add_measurements(
         self,
         data: pd.DataFrame,
-        numerical_measurements_must_be_within_tolerance: bool = True,
+        on_tolerance_violation: Literal["raise", "warn", "ignore"] = "raise",
```

Comment:
since these violations are not an issue for scaling,  I would go with `warn` as default, not `raise`. This is also needed in the API I think.

The user can explicitly still disallow violations but setting `raise` which would be similar to explicitly setting the old `numerical_measurements_must_be_within_tolerance` to `True`

Reply:
Agree.